### PR TITLE
Add info on how to edit quickstart files within lando containers with…

### LINKED ISSUE #63: Add info about IDE integration into docker container directories.
## Motivation
As a developer, I would like to benefit from code hinting and a better editing experience overall.

### Is your feature request related to a problem? Please describe.
Code editing is done within a docker container, which is often inaccessible to IDE's like sublime text or even vim.

### Describe the solution you'd like
Add info about IDE integration into docker container directories.

### PR DESCRIPTION
**Pull request by trackleft**
… Visual Studio Code (VS Code)

## Description
Closes #63  

## Types of changes
- [x] Documentation

### CODE REVIEW

**Comment by joeparsons** [comment]
PHPCS doesn't like the 2 blank lines at the end of CONTRIBUTING.md.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,19 +12,26 @@ team of web-focused volunteers that meet weekly to create projects like [Arizona
...
       to Arizona Digital meetings, both physical and via Zoom.
     * Use the `#uadigital-general` channel to ask general questions related to
       Arizona Digital.
   * A basic understanding of [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
-  * [Lando](https://docs.lando.dev/basics/installation.html) or [DDEV](https://www.ddev.com/get-started/) for local development.
+  * Local containerized (Docker) development environment tool either lando or ddev
+    * [Lando](https://docs.lando.dev/basics/installation.html)
+    * [DDev](https://docs.lando.dev/basics/installation.html)
+  * An IDE with the capability to attach to a remote codeserver or docker
+    container. [Visual Studio](https://visualstudio.microsoft.com) allows for this.
```
Comment (camikazegreen) [inline CONTRIBUTING.md:24]: Visual Studio Code

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -95,4 +104,34 @@ ddev config --project-type php
...
+want to ssh into a docker container and use nano, or rsync code from local into
+the docker container 100 times.
+
+Note: These instructions may not work for DDEV.
+
+Setup:
+  1. Make sure you have [Visual Studio Code installed](https://code.visualstudio.com/docs/introvideos/basics),
+  2. Install the Visual Studio Code [Visual Studio Code Remote Development Extension Pack
+](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+  3. In the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette),
```
Comment (trackleft) [inline CONTRIBUTING.md:123]: Command Palette is found in View menu (⌘⇧P on Mac)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -95,4 +104,34 @@ ddev config --project-type php
...
+    type `Remote-containers: Attach to Running Container...` and click on the
+    result to bring up a list of running containers on your computer.
+  4. If using lando, find the container whose name ends with `_appserver_1` and click
+    it to attach to that container.
+  5. The last thing you'll have to do is add a folder from within the container
+    to your workspace.  To do this, use the file menu within Visual Studio Code,
+    to Add Folder to Workspace.  Then add the app folder and you are ready to
```
Comment (trackleft) [inline CONTRIBUTING.md:130]: Then replace /root with /app

File: CONTRIBUTING.md (hunk 1 above)
Comment (joeparsons) [reply CONTRIBUTING.md]: Maybe https://code.visualstudio.com instead of https://visualstudio.microsoft.com ?

Comment (trackleft) [reply CONTRIBUTING.md]: I've updated, thanks for the link!

**Comment by tadean** [comment]
I tested out the included instructions and was able to successfully connect to the running container with Visual Studio Code on Linux, with access to the Drupal codebase inside the container. Nice!

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -12,19 +12,26 @@ team of web-focused volunteers that meet weekly to create projects like [Arizona
...
-    * Use the `#friday-meetings` channel to ask questions or get updates related 
+    * Use the `#friday-meetings` channel to ask questions or get updates related
       to Arizona Digital meetings, both physical and via Zoom.
     * Use the `#uadigital-general` channel to ask general questions related to
       Arizona Digital.
   * A basic understanding of [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
-  * [Lando](https://docs.lando.dev/basics/installation.html) or [DDEV](https://www.ddev.com/get-started/) for local development.
+  * Local containerized (Docker) development environment tool either lando or ddev
+    * [Lando](https://docs.lando.dev/basics/installation.html)
+    * [DDev](https://docs.lando.dev/basics/installation.html)
```
Comment (joeparsons) [inline CONTRIBUTING.md:22]: This needs to be a link to DDev instead of lando.

**Comment by joeparsons** [comment]
@joshuasosa Could you review the ddev changes in this PR when you have a chance?

**Pull request merged**